service/controller/key/key.go

```diff
@@ -0,0 +1,18 @@
+package key
+
+import (
+	"github.com/giantswarm/aws-operator/pkg/project"
+)
+
+func VersionLabelSelector(enable bool, overrideVersion string) string {
+	if !enable {
+		return ""
+	}
+
+	version := project.Version()
+	if overrideVersion != "" {
+		version = overrideVersion
+	}
+
+	return project.Name() + ".giantswarm.io/version=" + version
```

Comment:
This should come from `pkg/label` where we have the operator version label already. Using the `project` package for the operator name would be cool though.

Reply:
I'll use `label.OperatorVersion` but I don't want to use `project.Name()` there because it will have to become a variable. Unless we switch to functions there.

Reply:
Done.